types/ember/test/ember-module-tests.ts

```diff
@@ -43,7 +43,7 @@ Ember.get({ z: 23 }, 'zz'); // $ExpectType unknown
 // getEngineParent
 Ember.getEngineParent(new Ember.EngineInstance()); // $ExpectType EngineInstance
 // getOwner
-Ember.getOwner(new Ember.Component());
+Ember.getOwner(new Ember.Component()); // $ExpectType Owner
```

Comment:
this works differently compared to underlying Glimmer, it can return undefined (at least this is how it reads in .ts def)

Reply:
This is how Ember works. 🙂